content/_index.md

```diff
@@ -34,15 +35,18 @@ By embedding the RSQkit and services into the EOSC Science Clusters, EVERSE will
 EVERSE ultimate ambition is to contribute towards a cultural change where research software is recognized as a first-class citizen of the scientific process and the people that contribute to it are credited for their efforts.
 
 EVERSE is coordinated by the [Centre for Research and Technology hellas (CERTH)](https://www.certh.gr). EVERSE will be interfacing with the [EOSC science clusters](https://eosc-portal.eu/esfri-thematic-cluster-projects) and their emerging use cases:
-
-{{< flex-content >}}
 - [ENVRI-FAIR](https://envri.eu/home-envri-fair/): Essential Climate Variables
 - [EOSC-Life](https://www.eosc-life.eu/): The Workflow Execution Service backend with RO-Crate
 - [ESCAPE](https://projectescape.eu/): Particle physics and astrophysics in the Dark Matter Science Project
 - [PaNOSC](https://www.panosc.eu/): Photon and neutron science through LEAPS/LENS
 - [SSHOC](https://sshopencloud.eu/): UDPipe language processing suite
-<--->
+
+<!-- {{< flex-content >}}
 {{< figure src="/images/logos/science-clusters_0.png" alt="Science Cluster Organisation Logo" >}}

```

Comment:
I rather liked this icon and the layout! Was it removed to support theme switching? (It is true that images with white backgrounds look bad when in dark mode).

Reply:
I also like the figure there. I removed it for two reasons mostly:
1. As you said going to the dark theme looks really bad
2. Also the png figure seems a little bit blurred (it doesn't have great resolution)

Reply:
OK, we drop it for now and we can try to get a better version (I was mainly experimenting with the 2-column layout, to be honest, and a screenshotted the image from their website!)